libdnf5/base/goal.cpp

```diff
@@ -1323,26 +1328,34 @@ void Goal::Impl::add_rpms_to_goal(base::Transaction & transaction) {
 }
 
 
-void Goal::Impl::add_remove_to_goal(
+GoalProblem Goal::Impl::add_remove_to_goal(
     base::Transaction & transaction, const std::string & spec, GoalJobSettings & settings) {
     bool clean_requirements_on_remove = settings.resolve_clean_requirements_on_remove(base->get_config());
     rpm::PackageQuery query(base);
     query.filter_installed();
 
     auto nevra_pair = query.resolve_pkg_spec(spec, settings, false);
     if (!nevra_pair.first) {
-        transaction.p_impl->report_not_found(GoalAction::REMOVE, spec, settings, libdnf5::Logger::Level::WARNING);
-        return;
+        auto & cfg_main = base->get_config();
+        bool skip_unavailable =
+            settings.skip_unavailable == GoalSetting::AUTO ? true : settings.resolve_skip_unavailable(cfg_main);
+        auto problem = transaction.p_impl->report_not_found(
+            GoalAction::REMOVE,
+            spec,
+            settings,
+            skip_unavailable ? libdnf5::Logger::Level::WARNING : libdnf5::Logger::Level::ERROR;);
```

Comment:
The last change created an issue
```
/tmp/tito/rpmbuild-dnf50v0o56qt/BUILD/dnf5-git-2783.0c657ee/libdnf5/base/goal.cpp:1346:95: error: expected ')' before ';' token
 1346 |             skip_unavailable ? libdnf5::Logger::Level::WARNING : libdnf5::Logger::Level::ERROR;);
      |                                                                                               ^
      |                                                                                               )
/tmp/tito/rpmbuild-dnf50v0o56qt/BUILD/dnf5-git-2783.0c657ee/libdnf5/base/goal.cpp:1342:60: note: to match this '('
 1342 |         auto problem = transaction.p_impl->report_not_found(
      |                                                            ^
/tmp/tito/rpmbuild-dnf50v0o56qt/BUILD/dnf5-git-2783.0c657ee/libdnf5/base/goal.cpp:1346:96: error: expected primary-expression before ')' token
 1346 |             skip_unavailable ? libdnf5::Logger::Level::WARNING : libdnf5::Logger::Level::ERROR;);
      | 
```
Please remove `;` => `skip_unavailable ? libdnf5::Logger::Level::WARNING : libdnf5::Logger::Level::ERROR);`